src/borg/key.py

```diff
@@ -154,13 +226,17 @@ class AESKeyBase(KeyBase):
 
     PAYLOAD_OVERHEAD = 1 + 32 + 8  # TYPE + HMAC + NONCE
 
+    def __init__(self, repository):
+        super(AESKeyBase, self).__init__(repository)
```

Comment:
looks like this is not needed any more.

btw, python3 has argumentless super() for the usual case.